packages/gitbook/src/app/(space)/(core)/~gitbook/ogimage/[pageId]/route.tsx

```diff
@@ -1,53 +1,210 @@
+IvlyztbcF66ESABQAA9KLQB5jLMOO2AAsAAACAoQmwAACgPzOR8nCLGRHp03QnwAIAAABgaAIsAAA4hhkMALCQAAsAAOhFaAfAJgIsAAAAAIYmwAIAANiPWWcADQiwAAAAaCnTNy8KKPPwjaIHE2ABAADAdkINIhumfwuwAAAAABiaAAsAAABYyq10HEKABQAA/Sn+8tDWAA0IsAAA4BiCDSLTv4lGnz6YAAsAAACAp1jEHQAASMW6OQBsJsACAAAA1hBG050ACwAAAIChCbAAAABgGzORoBMBFgAAAACPDbV2oQALAADoJetsFbN0AK4kwAIAANiP8IqI9Gu6E2ABAAAAXCa0O5gACwAAgNayFPtCDehEgAUAAADA0ARYAAAAAAxNgAUAALAft5gRUcZ+7b18MAEWAAAArSn2gaYEWAAAQC9CDaI5Ff2a2Ibp3wIsAAAAAIYmwAIAAABgaAIsAAAAYI2si7i7XfRAAiwAAABaUugDzQmwAACAXoQaAHOxiDsAAAAwJWE03QmwAAAA9qPQB2hAgAUAAADA0ARYAAAAsI0F6/PI+s2Lw7gtAAAA+1PkA8zlpljEHQAAIAXBHTCrocYvARYAAACwhmCW7gRYAAAAtCbgAJoSYAEAAAAwNAEWAAAAAEMTYAEAAOzL7XTArHwLIQAAQALCK4AGBFgAAACwjYAyj1PR3ocSYAEAANBapkJfqAEdCLAAAACANYR2dCfAAgAAAJYSXnEIARYAAMB+FPsADQiwAAAAaMli10SkTx9MgAUAAADA0ARYAAAAADw21GxKARYAAACwhtvp6E6ABQAAAMDQBFgAAAC0lmWGjplIeWjrgwmwAAAAABiaAAsAAACApwwz8+y2AAAAcI2bC/93WvCcqFof88ivYabJIfVYT2W8Y67941T27Sev7x97GLF/v7p/DJEdZRxEAQAAWthSMK+twfao2bZuc+nP7fV6rH3ulvBwr/bZs933qusvbbeGGp9XPH/L89Y+d++fOQd3exmxj5zDuk+NtjlCvwYAAOAJGYuwWui/KTm8K3mO9ay27zcln2/vHm9LLrV//1AGYQ0sAAAAAIYmwAIAAABgaAIsAAAAAIYmwAIAANjPMF9B39Gp5DrurG0MXQmwAAAAAC7LFsyeDXPMAiwAAADYxkwk6ESABQAAAMDQBFgAAADAGmae0Z0ACwAAgNYEHESjTx9MgAUAAADA0ARYAAAAADzFtxACAAAAU3I7Hd0JsAAAAAAYmgALAAAAgMeGmmknwAIAAIBtTsXtdMRmDSwAAAAAWEKABQAAAKxh1hndCbAAAAAALhPaHUyABQAAQGuKfaApARYAAAAAQxNgAQAAADA0ARYAAAAAQxNgAQAAADA0ARYAAACtZVnE/VQsWJ+Fdj6YAAsAAACApwwT3AmwAAAAgKXMROIQAiwAAABgDSEW3d0WAAAAaEe4QUTWOzuYGVgAAAAAPMUaWAAAAACwhAALAAAAWMOtdHQnwAIAAABgaAIsAAAAgMvMOjuYAAsAAAC2EWoQ2VD9W4AFAAAAwFN8CyEAAAAALCHAAgAAANZw6yTdCbAAAAAAGJoACwAAALYxEwk6EWABAAAAawju8rCIOwAAAAAsIcACAAAAuMyss4MJsAAAAGAboUYu2vtAAiwAAADYTqgBHQiwAAAAABiaAAsAAABYw6wzuhNgAQAAAFwmtDuYAAsAAACAx4YK7QRYAAAAADxlmBBLgAUAAADA0ARYAAAAwBrWg6I7ARYAAABsI8jJQ1sfTIAFAAAA29wUwQZ0IcACAACAbYRXRGcRdwAAAGBKgju6E2ABAAAAMDQBFgAAAABDE2ABAAAAMDQBFgAAAABDE2ABAAAAMDQBFgAAAABDuy0AAADAqG7KmEbdr2tcOqbT/f9nmgh0fj2GOOaIHQ4AAAAiWVu7L33+1kyg/tzphf/fss2Wz9v6/Od+5rlj3mN/Hv6uPV6XLds8lbK5v7y07ZbPBQAAAPglTMh4N9e7km8ZptrOH8sgrIEFAAAALHUql2dfwS4EWAAAAAAMTYAFAAAAcJlZZwcTYAEAAADwlGGCOwEWAAAAAEMTYAEAAABruJ2O7gRYAAAAAAxNgAUAAADAY6diDSwAAAAAWEaABQAAAMDQBFgAAADAGhkXcbdw/cEEWAAAAACX3RQOJcACAAAAuMwMrIMJsAAAAAAYmgALAAAAgKEJsAAAAIA1LOJOdwIsAAAAAIYmwAIAAADgKcPMPBNgAQAAADA0ARYAAAAAQxNgAQAAAFx2KhZyP5QACwAAAIChCbAAAAAAeIpF3AEAAABgCQEWAAAAAEMTYAEAAAAwNAEWAAAAwGUZv4FwqGMWYAEAAADwFIu4AwAAAMASAiwAAAAAhibAAgAAAGBoAiwAAAAAhibAAgAAAGBoAiwAAAAAnuJbCAEAAABgCQEWAAAAwGWnMtBspIwEWAAAAAAMTYAFAAAAwNAEWAAAAAAMTYAFAAAAwNAEWAAAAACXWcD9YAIsAAAAAB4b6psXBVgAAAAADE2ABQAAAMDQBFgAAAAADE2ABQAAAMDQBFgAAAAAPMUi7gAAAACwhAALAAAAgKEJsAAAAAAYmgALAAAAgKEJsAAAAAAYmgALAAAAgKEJsAAAAAAYmgALAAAAgKEJsAAAAAB4yqkMQoAFAAAAwNAEWAAAAAAMTYAFAAAAwNAEWAAAAAAMTYAFAAAAwNAEWAAAAAA8ZZhvIbwpAAAAAGS0Nhca7fkAAAAAJHZTBEoAAAAADG6YAMsaWAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQHD/AagXIjjIr64XAAAAAElFTkSuQmCC'
```

Comment:
Should we move these as variables?

or event better, either:
- Store them as images and import them (I don't know if next supports this out of the box)
- Put them in the public folder and reference by url (if it works?)

Reply:
I first had them as images in the public folder and that worked, but we (purposely?) have `/public/~gitbook/static/` in our `.gitignore` so it seemed cleaner to keep that as-is and also save us the round trip of fetching the image. But I agree it’s probably cleaner to have them in there, I’d just have to change that gitignore to store them in the right place.

Reply:
Update: proposing to move these images into `/src/images` and import them as you suggested.

Since these images should only be used as part of the ogimage generation process, it seems incorrect to host them in `/public/~gitbook/static/images` and so they should be part of `/src` instead.